src/scene_battle_rpg2k3.cpp

```diff
@@ -1318,10 +1370,18 @@ void Scene_Battle_Rpg2k3::Escape(bool force_allow) {
 
 	SetState(State_SelectActor);
 	ShowNotification(ToString(lcf::Data::terms.escape_failure));
+	SetWait(10, 30);
```

Comment:
There is also a `Wait(10, 30)` on escape success with no message, immediately after playing the sound effect.

Reply:
Added.